cmd/datasource-syncer/datasource-syncer.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: datasource-syncer
+spec:
+  schedule: 0 * * * * # run once an hour, must run at least once an hour.
```

Comment:
Feels like running every hour could leave a brief moment of time where queries fail due to not being authed, just due to cron precision. What's the harm in doing every 30 minutes?